ethereum/contracts/interfaces/relayer/IWormholeRelayerTyped.sol

```diff
@@ -424,22 +452,32 @@ interface IWormholeRelayerSend is IWormholeRelayerBase {
      * (e.g. with a different delivery provider)
      *
      * This function must be called with `msg.value` equal to 
-     * quoteEVMDeliveryPrice(targetChain, newReceiverValue, newGasLimit, newDeliveryProviderAddress)
+     * quoteEVMDeliveryPrice(targetChain, newReceiverValue, newGasLimit, newDeliveryProviderAddress, verifyDeliveryVaa)
      * 
      * @param deliveryVaaKey VaaKey identifying the wormhole message containing the 
      *        previously published delivery instructions
      * @param targetChain The target chain that the original delivery targeted. Must match targetChain from original delivery instructions
      * @param newReceiverValue new msg.value that delivery provider should pass in for call to `targetAddress` (in targetChain currency units)
      * @param newGasLimit gas limit with which to call `targetAddress`. Any units of gas unused will be refunded according to the  
      *        `targetChainRefundPerGasUnused` rate quoted by the delivery provider, to the refund chain and address specified in the original request
+     * @param verifyDeliveryVaa If true, the delivery VAA will be verified on the target chain prior to calling targetAddress
+     *        Note: If this is false, targetAddress must implement IWormholeReceiverUnsafe instead of IWormholeReceiver,
+     *        and the IWormholeReceiverUnsafe.receiveWormholeMessagesUnsafe endpoint will be called instead!
      * @param newDeliveryProviderAddress The address of the desired delivery provider's implementation of IDeliveryProvider
      * @return sequence sequence number of published VAA containing redelivery instructions
+     * 
+     * @notice *** This will only be able to succeed if the following is true **
+     *         - newGasLimit >= gas limit of the old instruction
+     *         - newReceiverValue >= receiver value of the old instruction
+     *         - newDeliveryProvider's `targetChainRefundPerGasUnused` >= old relay provider's `targetChainRefundPerGasUnused`
+     *         - (verifyDeliveryVaa, oldVerifyDeliveryVaa) is not (false, true)
      */
     function resendToEvm(
         VaaKey memory deliveryVaaKey,
         uint16 targetChain,
         TargetNative newReceiverValue,
         Gas newGasLimit,
+        bool verifyDeliveryVaa,
```

Comment:
What's the usecase for allowing this to be changed by a resend? Is it safer to require the original send to specify whether it's verified or not? 

Is there a reason this has to be an option here?